src/main/java/org/killbill/billing/client/model/gen/BlockPrice.java

```diff
@@ -36,20 +36,20 @@ public class BlockPrice {
 
     private String unitName = null;
 
-    private Double size = null;
+    private BigDecimal size = null;
```

Comment:
That part however is surprising, it looks like the code regeneration did not take into account [all the changes](https://github.com/killbill/killbill-client-java/pull/97). \cc @xsalefter

Reply:
You're right, sorry. Probably because I thought It was different thing (`BlockPrice` vs `BlockPriceOverride`).